chapter_attention-mechanisms/attention-cues.md

```diff
@@ -200,8 +205,20 @@ we consider a simple case where
 the attention weight is one only when the query and the key are the same; otherwise it is zero.
 
 ```{.python .input}
-#@tab all
-attention_weights = d2l.eye(10).reshape(1, 1, 10, 10)
+#@tab mxnet
+attention_weights = np.eye(10).reshape(1, 1, 10, 10)
+show_heatmaps(attention_weights, xlabel='Keys', ylabel='Queries')
+```
+
+```{.python .input}
+#@tab pytorch
+attention_weights = torch.eye(10).reshape(1, 1, 10, 10)
+show_heatmaps(attention_weights, xlabel='Keys', ylabel='Queries')
+```
+```{.python .input}
+#@tab tensorflow
+eye_matrix= tf.eye(10)
```

Comment:
There seems no need to keep the `eye_matrix` variable. We may use

`d2l.reshape(d2l.eye(10), (1, 1, 10, 10))`

Reply:
I'll add a commit for the same then.